sleuth_backend/views/views_utils.py

```diff
@@ -75,6 +75,21 @@ def build_search_query(core, query_str, base_kwargs):
         kwargs['default_field'] = 'name'
         kwargs['highlight_fields'] = 'description'
 
+    elif core == 'redditPost':
+        fields = {
+            'id': 1,
+            'name': 7,
+            'description': 10,
+            'comments': 6,
+        }
+        query = Query(query_str) \
+            .fuzz(1)
```

Comment:
No need to put the call to `fuzz` on a new line since you're not chaining a bunch of calls.